--- libidn-0.4.9/java/Makefile.am.orig	2004-06-11 18:06:58.000000000 +0200
+++ libidn-0.4.9/java/Makefile.am	2004-06-20 18:21:06.865356816 +0200
@@ -41,7 +41,8 @@
 
 INSTALL_TARGETS += install-jar
 install-jar:
-	ln -sf libidn-@VERSION@.jar $(libidn_jardir)/libidn.jar
+	install -d $(DESTDIR)$(libidn_jardir)/libidn.jar
+	ln -sf libidn-@VERSION@.jar $(DESTDIR)$(libidn_jardir)/libidn.jar
 
 clean-local:
 	rm -f $(libidn_jar_DATA)
